config.ru

```diff
@@ -1,17 +1,17 @@
 require_relative 'plugin.rb'
 require_relative 'app/logging.rb'
 
+$stdout.sync = $stderr.sync = true
+
 if ENV['LOG_LEVEL']
   log_level = ENV['LOG_LEVEL'].to_i
 else
   log_level = Logger::INFO
 end
-Logging.initialize_logger(STDOUT, log_level)
+Logging.initialize_logger(STDERR, log_level)
```

Comment:
> Instead, each running process writes its event stream, unbuffered, to `stdout`

I might disagree with a too strict literal interpretation of that, but I'm not going to start arguing it here.

Removed the logging changes from this PR.